tgmg_test.go

```
@@ -0,0 +1,648 @@
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Setup command with flags similar to setWebhookCmd
+			cmd := &cobra.Command{}
+			cmd.Flags().String("url", "", "HTTPS URL to send updates to")
+			cmd.Flags().String("ip-address", "", "Fixed IP address")
+			cmd.Flags().Int("max-connections", 0, "Maximum connections")
+			cmd.Flags().String("secret-token", "", "Secret token")
+
+			// Set flags
+			for key, value := range tt.flags {
+				cmd.Flags().Set(key, value)
```

golangci-lint (1.64.8)

Error return value of `(*github.com/spf13/pflag.FlagSet).Set` is not checked

(errcheck)